extensions/extension.go

```diff
@@ -71,25 +39,28 @@ func initInternal() error {
 		MaxArgs:  1,
 		ArgTypes: []object.Type{object.FLOAT},
 	}
-	for _, function := range []OneFloatInOutFunc{
-		math.Sin,
-		math.Cos,
-		math.Tan,
-		math.Log, // renamed to ln in MathFunctionName
-		math.Sqrt,
-		math.Exp,
-		math.Asin,
-		math.Acos,
-		math.Atan,
-		math.Round,
-		math.Trunc,
+	for _, function := range []struct {
+		fn   OneFloatInOutFunc
+		name string
+	}{
+		{math.Sin, "sin"},
+		{math.Cos, "cos"},
+		{math.Tan, "tan"},
+		{math.Log, "ln"}, // proper name for natural logarithm and also doesn't conflict with logger builtin.
+		{math.Sqrt, "sqrt"},
+		{math.Exp, "exp"},
+		{math.Asin, "asin"},
+		{math.Acos, "acos"},
+		{math.Atan, "atan"},
+		{math.Round, "round"},
```

Comment:
I would add floor and ceil

Reply:
I think round and trunc are enough but sure, adding

Reply:
added